SMLHelper/Patchers/LargeWorldStreamerPatcher.cs

```diff
@@ -0,0 +1,56 @@
+using System.Collections.Generic;
+using System.IO;
+using HarmonyLib;
+using SMLHelper.V2.Handlers;
+using SMLHelper.V2.MonoBehaviours;
+using UnityEngine;
+
+namespace SMLHelper.V2.Patchers
+{
+    [HarmonyPatch(typeof(LargeWorldStreamer))]
```

Comment:
leaving it wont do anything but its redundant.

Reply:
0e57299